src/app/models/resources/internal-search-query.ts

```diff
@@ -0,0 +1,8 @@
+export interface InternalSearchQuery {
```

Comment:
License file, please add it. Same goes for the other new files. There is no automated process for this yet unfortunately.

Reply:
IntelliJ can do this for you 😏

Reply:
Are you getting paid by @Jetbrains ? ;)